picard/tagger.py

```diff
@@ -1246,12 +1246,58 @@ def sighandler(self):
         self.signalnotifier.setEnabled(True)
 
 
-def version():
-    print("%s %s %s" % (PICARD_ORG_NAME, PICARD_APP_NAME, PICARD_FANCY_VERSION_STR))
+class PicardArgs:
 
+    def __init__(self, argparse_args=None):
+        if argparse_args is not None:
+            self.load_from_argparse(argparse_args)
 
-def longversion():
-    print(versions.as_string())
+    def load_from_argparse(self, argparse_args):
+        self.CONFIG_FILE = argparse_args.config_file
+        self.DEBUG = argparse_args.debug
+        self._EXEC = argparse_args.exec
+        self.NO_PLAYER = argparse_args.no_player
+        self.NO_RESTORE = argparse_args.no_restore
+        self.NO_PLUGINS = argparse_args.no_plugins
+        self.NO_CRASH_DIALOG = argparse_args.no_crash_dialog
+        self.STAND_ALONE_INSTANCE = argparse_args.stand_alone_instance
+        self.VERSION = argparse_args.version
+        self.LONG_VERSION = argparse_args.long_version
+        self._FILE_OR_URL = argparse_args.FILE_OR_URL
+        self.PROCESSABLE = []
```

Comment:
Why capitalized attribute's names?

I have the feeling we could subclass `argparse_args` object and extends it, rather than copying all of its attributes.

Reply:
>Why capitalized attribute's names?
To show they're constant


>I have the feeling we could subclass argparse_args object and extends it, rather than copying all of its attributes.
I think it would compound the readability and simplicity of this class

Reply:
Even though there is no strict definition of a constant in Python (well, they are _usually_ defined at module level), those are normal class attributes, so I don't think they should be capitalized.
A class constant can be capitalized, but it is usually never modified, that's not the case here for `PROCESSABLE` which is initialized then modified.
I don't think upper-cased names add anything here.